src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
```

Comment:
As best a practice include the component name in the Prop type name `TodoInfoProps`; this helps avoid confusion and redefinition of types in larger projects